source/shared/cpp/ObjectModel/UnknownElement.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "pch.h"
+#include "BaseCardElement.h"
+#include "Enums.h"
+#include <time.h>
+#include "ElementParserRegistration.h"
+#include "DateTimePreparser.h"
+
+namespace AdaptiveCards
+{
+class UnknownElement: public BaseCardElement
+{
+public:
+    UnknownElement();
+    UnknownElement(Json::Value const &paylaod);
+        
+    virtual Json::Value SerializeToJsonValue() override;
+    Json::Value GetJsonPayload();
+    void SetJsonPayload(const Json::Value& payload);
```

Comment:
Do you need this, or does AdditionalProperties work for this?

Reply:
I see, this is just calling AdditionalProperties. You don't need to expose this, the caller can just call AdditionalProperties.

---
In reply to: [172369640](https://github.com/Microsoft/AdaptiveCards/pull/1208#discussion_r172369640) [](ancestors = 172369640)

Reply:
yes, we don't need this; thanks


---
In reply to: [172369640](https://github.com/Microsoft/AdaptiveCards/pull/1208#discussion_r172369640) [](ancestors = 172369640)

Reply:
You don't need the getter either.  The caller can call GetAdditionalProperties.

---
In reply to: [172370738](https://github.com/Microsoft/AdaptiveCards/pull/1208#discussion_r172370738) [](ancestors = 172370738,172369640)

Reply:
doesn't sound weird to call getAdditonalProperty on unknown object?

---
In reply to: [172372405](https://github.com/Microsoft/AdaptiveCards/pull/1208#discussion_r172372405) [](ancestors = 172372405,172370738,172369640)